From 2fd2c44240f32423c040f909de2a323d72d4fe82 Mon Sep 17 00:00:00 2001
From: panchenbo <panchenbo@uniontech.com>
Date: Wed, 9 Sep 2020 19:48:57 +0800
Subject: [PATCH] fix compile error

---
 CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a50f11e..0f47e79 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -147,14 +147,14 @@ set(WIDGETS_SRCS
     ./src/widgets/fullscreenbackground.h
     ./src/widgets/userbutton.h
     ./src/widgets/controlwidget.h
-    ./src/widgets/errortooltip.h
+    #./src/widgets/errortooltip.h
     ./src/widgets/shutdownwidget.h
     ./src/widgets/dbreathinglabel.h
     ./src/widgets/rounditembutton.h
     ./src/widgets/kblayoutwidget.h
     ./src/widgets/mediawidget.h
     ./src/widgets/propertygroup.h
-    ./src/widgets/darrowrectangle.h
+    #./src/widgets/darrowrectangle.h
     ./src/widgets/userbreathinglabel.h
     ./src/widgets/virtualkbinstance.h
     ./src/widgets/logowidget.h
@@ -164,8 +164,8 @@ set(WIDGETS_SRCS
     ./src/widgets/fullscreenbackground.cpp
     ./src/widgets/loginbutton.cpp
     ./src/widgets/dbreathinglabel.cpp
-    ./src/widgets/errortooltip.cpp
-    ./src/widgets/darrowrectangle.cpp
+    #./src/widgets/errortooltip.cpp
+    #./src/widgets/darrowrectangle.cpp
     ./src/widgets/mediawidget.cpp
     ./src/widgets/controlwidget.cpp
     ./src/widgets/virtualkbinstance.cpp
-- 
2.20.1

